CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0
+
+- Rework API based on `flutter_callkit_incoming`'s API, add iOS implementation and fix bugs (BREAKING CHANGE)
+
```

Comment:
Not sure if we're ready for a stable `1.0` release yet 😅 I think let's go for a `0.3` and later we can promote it to `1.0` when we feel it's stable.

Reply:
I went for a Major bump to indicate backward incompatibility rather than stability but I see your point 😅

Reply:
Should probably move changelog to GitHub releases